webpack.config.js

```diff
@@ -9,6 +9,7 @@ var base = {
         host: '0.0.0.0',
         port: process.env.PORT || 8073
     },
+    devtool: 'source-map',
```

Comment:
Not sure about this change.  [Webpack's documentation](https://webpack.github.io/docs/configuration.html#devtool) on the various devtools is decidedly confusing — I can't tell what the default is to know what this is being changed from.  One thing is clear though, that `source-map` is the slowest option of all.  I haven't had too much of an issue with the current devtool, since at least in Chrome, I'm always able to figure out where the error is occurring.  As far as I can tell from googling and the docs, the recommended choice is `cheap-module-eval-source-map` for development.